client/src/components/screen/ProfileUpdate.tsx

```diff
@@ -96,6 +96,7 @@ const profileUpdate = graphql`
       name
       nickname
       statusMessage
+      photoURL
```

Comment:
Do you think `thumbURL` should be put in there as well?

Reply:
Yes!

Reply:
sorry, that content is just a response of the type, not to update 😆.
so I rolled back.